Extensions/PlatformBehavior/platformerobjectruntimebehavior.js

```diff
@@ -861,6 +862,14 @@ gdjs.PlatformerObjectRuntimeBehavior.prototype.getMaxFallingSpeed = function() {
   return this._maxFallingSpeed;
 };
 
+/**
+ * Get the maximum speed of Climbing the ladder
+ * @returns {number} The current speed of climbing the ladder
```

Comment:
This is a speed, not a current speed. "Current speed" would be to get the speed of the object

Reply:
got it.